src/Tizen.WebView/Tizen.WebView/SecurityOrigin.cs

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.ComponentModel;
+
+namespace Tizen.WebView
+{
+    /// <summary>
+    /// This class provides the properties for Security Origin of WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class SecurityOrigin
```

Comment:
No way to get or create SecurityOrigin object
This PR ia not completed

Reply:
SecurityOrigin is used to handle geolocation and user media permissions, but we have decided not to provide those features because those features have been internalized.
@BasavarajPS , please close this PR.

Reply:
> 
> 
> SecurityOrigin is used to handle geolocation and user media permissions, but we have decided not to provide those features because those features have been internalized.
> @BasavarajPS , please close this PR.

I close it, because it have been internalized